sdk/python/kfp/compiler/compiler.py

```diff
@@ -852,6 +853,10 @@ def _create_workflow(self,
     from ._data_passing_rewriter import fix_big_data_passing
     workflow = fix_big_data_passing(workflow)
 
+    output_directory = getattr(pipeline_func, 'output_directory', None)
```

Comment:
Can this be `pipeline_root` instead, to be consistent with the IR and the standard terminology?

Reply:
According to [IR](https://github.com/kubeflow/pipelines/blob/c8c9905bf2b2662a851c339afbd143cfdbdb603c/api/v2alpha1/pipeline_spec.proto#L36) it's called `gcs_output_directory ` currently, and it seems like `root` is now referring to the root component spec in the pipeline https://github.com/kubeflow/pipelines/blob/c8c9905bf2b2662a851c339afbd143cfdbdb603c/api/v2alpha1/pipeline_spec.proto#L88

WDYT?

Reply:
Discussed offline. Will go with `output_directory` for now for consistency.